index.html

```diff
@@ -296,61 +295,5 @@ <h2 class="heading">Contact <span>Us</span></h2>
             @ UniApplyBot | All Rights Reserved!
         </p>
     </footer>
-    <script>
```

Comment:
All the read section is the code I have taken into a file script.js and link it on the HTML codes so it can apply to the HTML file.